fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/InstrumentedReadTransaction.java

```diff
@@ -192,6 +194,23 @@ public AsyncIterable<KeyValue> getRange(Range range, int limit, boolean reverse,
         return new ByteCountingAsyncIterable(underlying.getRange(checkKey(range), limit, reverse, streamingMode));
     }
 
+    @Override
+    public AsyncIterable<MappedKeyValue> getMappedRange(final KeySelector begin, final KeySelector end, final byte[] mapper, final int limit, final boolean reverse, final StreamingMode mode) {
+        increment(FDBStoreTimer.Counts.READS);
+        increment(FDBStoreTimer.Counts.RANGE_READS);
+        return underlying.getMappedRange(begin, end, mapper, limit, reverse, mode);
+    }
+
+    @Override
```

Comment:
Do we know for certain that no counters should be incremented for these calls? (for getMappedRange work has been done on the index prefetch branch, but I am not aware of similar work for the getRangeSplitPoints)

Reply:
I'm...not sure, actually. It looks like that method probably uses the byte sample to get an estimate for relatively equally sized key ranges. My instinct was to treat these like `getEstimatedRangeSizeBytes` as of a similar type, and leave it uninstrumented. That being said, I'm not opposed to adding instrumentation here, either in this PR or in a follow up.

Reply:
Sure. We can leave uninstrumented and change in followup PR.